models/cable/R/write.config.CABLE.R

```diff
@@ -0,0 +1,125 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the 
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+
+##-------------------------------------------------------------------------------------------------#
+##' Writes a CABLE config file.
+##'
+##' Requires a pft xml object, a list of trait values for a single model run,
+##' and the name of the file to create
+##'
+##' @name write.config.CABLE
+##' @title Write CABLE configuration files
+##' @param defaults list of defaults to process
+##' @param trait.samples vector of samples for a given trait
+##' @param settings list of settings from pecan settings file
+##' @param run.id id of run
+##' @return configuration file for CABLE for given run
+##' @export
+##' @author Rob Kooper
```

Comment:
Did  you add any code to this function or is it just the default template? If the former you should add your name